### DISCUSSION TITLE
No response from GitHub Support

### DISCUSSION THREAD

**Discussion by JorgeGarciaEnki** [community/community#144694] [Enterprise] [answered]
### Select Topic Area

General

### Body

Hello,
Does anyone know how to get the support team's attention?
I raised a ticket about 15 days ago and no one has responded to it.
I have seen in the forum that they then take months to respond but how is it possible for that to happen?

Any Clue?

Thanks.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @JorgeGarciaEnki, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
